packages/e2e-tests/src/specs/editor/fontCheck/fontCheck.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  withExperimentalFeatures,
+  createNewStory,
+  visitAdminPage,
+  insertStoryTitle,
+  publishStory,
+  takeSnapshot,
+  visitSettings,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Font Check', () => {
+  beforeEach(() => () => {});
+
+  afterEach(async () => {});
+
+  describe('Font Check Dialog', () => {
+    withExperimentalFeatures(['customFonts']); // 'notifyDeletedFonts'
```

Comment:
@todo - Passing 2 feature here seems to cause some issues

Reply:
Have you tried incorporating the change from https://github.com/GoogleForCreators/web-stories-wp/pull/7748/files#diff-043d824399de44fd91ca884ab3d8564fbde22f0bc88138c0b9282c1c897d6641 as suggested?

Reply:
Yes tried that --- will do a more focussed look at just that piece today.

Reply:
Happy to take a look tomorrow or so if needed